### PR TITLE
Make alpine-linux a seperate os

### PR DESCRIPTION
**Pull request by johnoliver**
# Checklist

- [ ] You added tests to cover the change
- [ ] `mvn clean install` build and test completes
- [ ] You changed or added to the documentation

### CODE REVIEW

**Comment by gdams** [comment]
@johnoliver test failures :(

File: adoptopenjdk-api-v3-models/src/main/kotlin/net/adoptopenjdk/api/v3/models/OperatingSystem.kt [hunk 1]
```diff
@@ -3,13 +3,14 @@ package net.adoptopenjdk.api.v3.models
...
+@Schema(type = SchemaType.STRING, enumeration = ["linux", "windows", "mac", "solaris", "aix", "alpine-linux"])
 enum class OperatingSystem : FileNameMatcher {
-    linux(0, "LinuxLH", "alpine-linux"),
+    linux(0, "LinuxLH"),
     windows(0, "win"),
     mac,
     solaris,
-    aix;
+    aix,
+    `alpine-linux`;
```
Comment (karianna) [inline adoptopenjdk-api-v3-models/src/main/kotlin/net/adoptopenjdk/api/v3/models/OperatingSystem.kt:13]: whitespace?

**Review by karianna — APPROVED** [review]
lgtm

**Review by gdams — APPROVED** [review]

File: adoptopenjdk-api-v3-models/src/main/kotlin/net/adoptopenjdk/api/v3/models/OperatingSystem.kt (hunk 1 above)
Comment (johnoliver) [reply adoptopenjdk-api-v3-models/src/main/kotlin/net/adoptopenjdk/api/v3/models/OperatingSystem.kt:13]: the whitespace is on, note there is a backtick there

**Pull request merged**